samples/apps/logging/logging.cpp

```diff
@@ -52,6 +52,7 @@ namespace loggingapp
   class CustomAuthPolicy : public ccf::AuthnPolicy
   {
   public:
+    static constexpr auto SECURITY_SCHEME_NAME = "";
```

Comment:
```suggestion
    static constexpr auto SECURITY_SCHEME_NAME = "CustomAuthPolicy";
```

Since `SECURITY_SCHEME_NAME` is now part of the required API, we should use a sensible value for this sample policy.

Reply:
Done